### PR TITLE
[feature] adds .travis.yml when generating a new app

### PR DESCRIPTION
**Pull request by apaganobeleno**
Adds .travis.yml to the newly generated project as part of #36 

This also allows to avoid ci file being generated by using the `--ci-provider=none` flag

cc @markbates

### CODE REVIEW

**Comment by markbates** [comment]
I think I would prefer the default to be no CI and let people choose. Thoughts?

**Comment by apaganobeleno** [comment]
@markbates i'm ok with that, lets change it.

File: buffalo/cmd/app_generators.go [hunk 1]
```diff
@@ -361,3 +366,18 @@ ratings:
     - "**.go"
 
 `
+
+const nTravis = `language: go
+env:
+- GO_ENV=test
+
+before_script:
```
Comment (markbates) [inline buffalo/cmd/app_generators.go]: this raises an interesting question about databases here. this is specific to Postgres, but what if they generate with mysql or sqlite?

Comment (apaganobeleno) [inline buffalo/cmd/app_generators.go]: Oh! Good catch @markbates

Comment (apaganobeleno) [inline buffalo/cmd/app_generators.go]: i'll work on this soon and keep you posted.

**Comment by philipithomas** [comment]
Travis has a linter for their .travis.yml files - I think that it should be incorporated somewhere into testing for this

**Comment by apaganobeleno** [comment]
@philipithomas validating the .travis.yml sounds like a great idea, however would imply installing ruby inside the docker container as well as the travis gem, @markbates what do you think? should we add that validation ?

**Comment by markbates** [comment]
If we validate the file we're embedding here, I think that's good enough for now. Unless the have an API we can ping or something.

**Comment by apaganobeleno** [comment]
@markbates great, i don't know of an API for that, just to clarify, when you say validate, you mean we check in our tests the file being created and its content, right ? something else we should check on that validation ?

**Comment by apaganobeleno** [comment]
@markbates @philipithomas seems there is one :) https://docs.travis-ci.com/api#linting, i'll work on this one over the week.

**Comment by apaganobeleno** [comment]
As an update while using the yml linter i found an [issue](https://github.com/travis-ci/travis-yaml/issues/110) with the validator (at least the API) when using Curl.

**Comment by markbates** [comment]
@apaganobeleno is this ready to merge, or are you still working on it?

**Comment by apaganobeleno** [comment]
@markbates this one is ready to merge, was looking for a way to have some tests that could cover running the new command and passing the --ci-provider travis flag, but the GOPATH check on the command is making it hard, anyways we need to add some tests there so, i think we can cover that lately.

**Pull request merged**